core/http-auth-aws/pom.xml

```diff
@@ -81,6 +81,11 @@
             <version>${awsjavasdk.version}</version>
             <optional>true</optional>
         </dependency>
+        <dependency>
+            <groupId>software.amazon.eventstream</groupId>
+            <artifactId>eventstream</artifactId>
+            <version>${eventstream.version}</version>
+        </dependency>
```

Comment:
This should have been an optional dependency like http-auth-aws-eventstream above, so all customers of http-auth-aws don't bring in eventstream unless they are using event streaming features. Fixing in https://github.com/aws/aws-sdk-java-v2/pull/4574.